src/recordlinker/main.py

```diff
@@ -139,8 +139,9 @@ async def health_check(db_session: orm.Session = Depends(get_session)) -> Health
 
 
 # Sample requests and responses for docs
-sample_link_record_requests = utils.read_json_from_assets("sample_link_record_requests.json")
-sample_link_record_responses = utils.read_json_from_assets("sample_link_record_responses.json")
+# TODO: These assets need to be installed with the python code
```

Comment:
What does this comment mean?

Reply:
Great question, so this is looking forward a bit to Dockerfile and pypi installation process.  Likely for both, but definitely for the latter, only the code in the "src/" directory is going to be installed.  So this code will fail as is because the sample files are outside of the "src/" directory.

Its not as simple as moving the `assets/` directory, because by default python is only going to install *.py files.  We'll need to update pyproject.toml to instruct the installation process to install these json files as well.